Automattic/wp-calypso
# Nav redesign: refactor: client/sites-dashboard-v2/{routes -> index}

## Proposed Changes

Renames `client/sites-dashboard-v2/{routes -> index}.tsx`

## Why are these changes being made?

To be consistent with existing routes in `client/` directory.

## Testing Instructions

1. Go to `/sites`
2. Verify that nothing breaks

## Pre-merge Checklist
